python-sdk/tests/sql/operators/test_load_file.py

```diff
@@ -427,14 +427,11 @@ def test_load_file_using_file_connection_fails_nonexistent_conn(
         "input_file": File(path=file_uri, conn_id=file_conn_id),
         "output_table": Table(name=OUTPUT_TABLE_NAME, **sql_server_params),
     }
-    with pytest.raises(BackfillUnfinished):
+    with pytest.raises(AirflowNotFoundException):
         with sample_dag:
             load_file(**task_params)
         test_utils.run_dag(sample_dag)
 
-    expected_error = "Failed to execute task: The conn_id `fake_conn` isn't defined."
-    assert expected_error in caplog.text
```

Comment:
Should we keep this error since `AirflowNotFoundException` can occur for few more cases?

<img width="599" alt="image" src="https://user-images.githubusercontent.com/8811558/193929767-69a7d0e4-a30c-4518-86c3-327d0c50f36d.png">

Reply:
@kaxil turns out you can actually do this with pytest.raises to check error message

```
    with pytest.raises(AirflowNotFoundException, match=r"The conn_id `fake_conn` isn't defined"):
        with sample_dag:
            load_file(**task_params)
        test_utils.run_dag(sample_dag)
```

Reply:
Yeah, go for it 🎉

Reply:
@kaxil added that. The other exceptions are simple things like "filenotfound" so we don't need to check error logs for those.